src/style.css

```
@@ -1 +1,37 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n + 1) {
```

```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
... {
```
Consider using commas to group your selectors